Spill of parallel aggregation in TiDB (#15755)
pingcap/docs: system-variables.md

- Default value: `OFF`
- This variable controls whether to enable TiDB to collect `PREDICATE COLUMNS`. After enabling the collection, if you disable it, the information of previously collected `PREDICATE COLUMNS` is cleared. For details, see [Collect statistics on some columns](/statistics.md#collect-statistics-on-some-columns).

> ### tidb_enable_concurrent_hashagg_spill <span class="version-mark">New in v7.6.0</span>

Reviewer: ```suggestion
### tidb_enable_concurrent_hashagg_spill <span class="version-mark">New in v8.0.0</span>
```

> ### tidb_enable_concurrent_hashagg_spill <span class="version-mark">New in v7.6.0 </span>

Reviewer 2: ```suggestion
### tidb_enable_concurrent_hashagg_spill <span class="version-mark">New in v7.6.0</span>
```

> ### `tidb_enable_concurrent_hashagg_spill` <span class="version-mark">New in v7.6.0 </span>

Reviewer: ```suggestion
### tidb_enable_concurrent_hashagg_spill <span class="version-mark">New in v7.6.0 </span>
```

> **Warning:**
>

> > Currently, it is an experimental feature. It is not recommended that you use it in production environments. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

Reviewer 2: ```suggestion
> Currently, the feature controlled by this variable is experimental. It is not recommended that you use it in production environments. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Boolean
- Default value: `ON`

> - This variable controls whether spill could be triggered in parallel HashAgg. When it's `ON`, parallel HashAgg will support the disk spill. This variable will be aborted when this feature is official released.

Reviewer: ```suggestion
- This variable controls whether TiDB supports disk spill for the concurrent HashAgg algorithm. When it is `ON`, disk spill can be triggered for the concurrent HashAgg algorithm. This variable will be deprecated when the feature is generally available in a future release.
```

Reviewer: 变量名里“并发”用的是  concurrent
